pkg/components/util/install.go

```diff
@@ -30,39 +30,47 @@ import (
 	"github.com/kinvolk/lokomotive/pkg/k8sutil"
 )
 
-func ensureNamespaceExists(name string, kubeconfig []byte) error {
+func ensureNamespaceExists(ns k8sutil.Namespace, kubeconfig []byte) error {
```

Comment:
It seems this function is reduntant with `UpdateNamespace` introduced before. I don't see a case where we would like to only update the namespace, but never create it. Can we merge those 2 functions?